bazel/repository_locations.bzl

```diff
@@ -1201,8 +1201,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Common Expression Language (CEL) C++ library",
         project_desc = "Common Expression Language (CEL) C++ library",
         project_url = "https://opensource.google/projects/cel",
-        version = "da0aba702f44a41ec6d2eb4bbf6a9f01efc2746d",
-        sha256 = "d62b93fd07c6151749e83855157f3f2778d62c168318f9c40dfcfe1c336c496f",
+        version = "e14d338d0f1bc64e54c4275ac5034fe5dedba969",
```

Comment:
same question here

Reply:
Same principle as abseil. We can ask them to cut a release, however.